EffortlessUI-new/public/base.css

```diff
@@ -0,0 +1,384 @@
+:root {
+  /* Colors */
+  --blue: #2874f0;
+  --pink: #ff3e6c;
+  --red: #dc3545;
+  --yellow: #e2b032;
+  --lightgreen: #42b72a;
+  --purple: #7952b3;
+  --lightOrange: #ff9f00;
+  --orange: #fb641b;
+  --greenishBlue: #17a2b8;
+  --grey: #f0e9e6;
+  --maroon: #cd486b;
```

Comment:
It's good that you had multiple colors but you can add primary and secondary colors for repetitive use